content/en/core/overview/architecture.md

```diff
@@ -18,11 +18,11 @@ A free and open source NoSQL database we use to store all our data, configuratio
 
 ### api
 
-A NodeJS service which runs on the server as a wrapper around CouchDB. It provides security and APIs for browsers and integrations. It also includes a custom implementation of filtered replication to allow it to support more concurrent users. See more at the [Medic API site](https://github.com/medic/medic/tree/master/api) on Github.
+A NodeJS service which runs on the server as a wrapper around CouchDB. It provides security and APIs for browsers and integrations. It also includes a custom implementation of filtered replication to allow it to support more concurrent users. See more at the [Medic API site](https://github.com/medic/cht-core/tree/master/api) on Github.
```

Comment:
```suggestion
A NodeJS service which runs on the server as a wrapper around CouchDB. It provides security and APIs for browsers and integrations. It also includes a custom implementation of filtered replication to allow it to support more concurrent users. See more at the [Medic API site](https://github.com/medic/cht-core/tree/main/api) on Github.
```

Reply:
Hey @mrjones-plip, thanks for checking all of this. In this case, I'm not changing "master" from "main" **yet** because the cht-core does not have a default main branch. I'm expecting cht-core to be the last one to be modified.

So, if I change it here, what would happen is that the link would go to a 404 (a non-existing branch). Instead, I plan to update this documentation again once all the other branches have been modified. The links will continue to work after that modification as GitHub will handle the redirects from master to main once the change is applied. I'll create a new pull request to avoid the 301 redirects.

Btw, this comment applies to all the other suggestions you pointed out here, as most of them were for cht-core, cht-conf, or some repositories that don't have a "main" branch yet.

Reply:
sounds like a plan!